cms/djangoapps/contentstore/views/import_export.py

```diff
@@ -273,11 +293,23 @@ def get_dir_for_fname(directory, filename):
 
             return JsonResponse({'Status': 'OK'})
     elif request.method == 'GET':  # assume html
-        course_module = modulestore().get_course(course_key)
-        return render_to_response('import.html', {
-            'context_course': course_module,
-            'successful_import_redirect_url': reverse_course_url('course_handler', course_key),
-            'import_status_url': reverse_course_url("import_status_handler", course_key, kwargs={'filename': "fillerName"}),
+        status_url = reverse_course_url(
+            "import_status_handler", course_key, kwargs={'filename': "fillerName"}
+        )
+        if library:
+            template = 'import_library.html'
```

Comment:
Why can't these templates be refactored/paramterized and shared between libraries and courses?